pkg/i2gw/providers/gce/notification.go

```diff
@@ -22,6 +22,6 @@ import (
 )
 
 func notify(mType notifications.MessageType, message string, callingObject ...client.Object) {
-	newNotification := notifications.Notification{Type: mType, Message: message, CallingObjects: callingObject}
+	newNotification := notifications.NewNotification(mType, message, callingObject...)
```

Comment:
In go, variadic arguments should be plural, so CallingObject should be callingObjects

Reply:
Also, notify is a func, should we call it noifyFunc?

Reply:
I ll change the CallingObject to Calling Objects.

I dont think it makes a difference whether we use notify or notifyFunc. Both can convey the same message. We dont put Func after every function i suppose.

Reply:
Fair. But most of the functions are not passed as params to different functions. Totally fine to keep it like this as well though